index.html

```diff
@@ -0,0 +1,129 @@
+                <ul class ="preNav-item">
+                    <li><a href=""><i></i><img src="./images/facebook.svg" alt=""></i></a></li>
+                    <li><a href=""><i><img src="./images/twitter.svg" alt=""></i></a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">Page</a></li>
+                    <li><a href="">Logout</a></li>
+                </ul>
+            </div>
+            <nav class="navbar">
+                <div id="logo"><a href="index.html"><img src="https://www.haitilibre.com/images-a/g-27982.jpg" width="160px" height="60px" alt="logoPage"></a></div>
+                <div class="Hamberger" id="icM"></div>
+                <div class="nav" id="navL">
+                    <ul class="nav-item">
+                        <li class="nav-list"><a href="about.html" class="nav-link" id="about">About </a></li>
+                        <li class="nav-list"><a href="" class="nav-link">Program</a></li>
+                        <li class="nav-list"><a href="" class="nav-link">Join</a></li>
+                        <li class="nav-list"><a href="" class="nav-link">Sponsor</a></li>
+                        <li class="nav-list"><a href="" class="nav-link">News</a></li>
+                        <li class="nav-list"><button type="button" id="nav-btn"> Get Tickets</button></li>
+                    </ul>
+                    <div class="nav-btn"></div>
+                </div>
+            </nav>
+        </header>
+        <section id="presentation">
+            <div class=cover>
+                <div class="presentation-greeting">
+                    <p class="hello">"Hello sharing world"</p>
+                    <h1 class="creativeText">7TH ANNUAL <br> HAITI TECH SUMMIT <br> 2023</h1>
+                </div>
+                <div class="presentation-desc">
+                    <p>Experience the conference wherever you are. Register now for online access. Tune in live for the keynotes and watch
+                    sessions on demand. Also be sure to join our event</p>
+                </div>
+                <div class="presentation-schedule">
+                    <p class="date">2023.03.27 (MON) ~ 29(WED)</p>
+                    <p class="adresse">
+                        @Global Virtual Event.
+                    </p>
+                </div>
+            </div>    
+        </section>
+        <section id="program">
+            <div class="main-program" >
+                <p>Main Program </p>
+                <div class="t-buttom"></div>
+            </div>
+            <div class="program-grid">
+                <div class="program-item">
+                    <i><img src="./images/program_icon_01 1.svg" alt="programIcon"></i>
+                    <p class="program-title">Lecture</p>
+                    <p class="program-text">Listen to the speakers from various countries about the messages of sharing
+                        and opening.</p>
+                </div>
```

Comment:
- [ ] kindly consider adding each of this `program item` in an article tag
this will help you full-fill the below requirement 👇 
> The HTML uses semantic tags, sections use `section`, articles use `article`, the nav and footer use `nav` and `footer`.